addon/components/new-osf-navbar/template.hbs

```diff
@@ -55,12 +55,9 @@
                                 <li>
                                     <a href="{{navLink.href}}" onclick={{action 'click' 'link' (concat 'Navbar - ' navlink.name)}}>
                                         {{#if (eq navLink.type 'addAPreprint')}}
-                                            <span class="hidden-xs hidden-sm">
+                                            <span>
```

Comment:
Small change, but if we aren't using classes in this span anymore we can just remove it.